src/Widget/ProfileRowFormWidget.cpp

```diff
@@ -142,3 +144,28 @@ RowFormWidget::SaveValueFileReader(unsigned i,
   Profile::Set(registry_key, new_value2);
   return true;
 }
+
+bool
+RowFormWidget::SaveValue(unsigned i,
+  const char *registry_key,
+  BrokenDate &value) const noexcept
+{
+  const DataFieldDate &df =
+    (const DataFieldDate&)GetDataField(i);
+  assert(df.GetType() == DataField::Type::DATE);
+
+  auto new_value = df.GetValue();
+
+  if (!new_value.IsPlausible())
```

Comment:
Explain this. This is treated as "value hasn't changed"..? For a nullable DateEntry?

Reply:
You are right: So you cannot clear a valid date - but if this necessary? A date of birth should be fixed - and never cleared... But I'm missing a BrokenDate::IsEmpty() - but on an invalide date it should change nothing...

Reply:
You didn't answer my question. This is not about whether clearing a date should be possible. I asked what this line of code is supposed to do, why it is here.